style.css

```diff
@@ -3,15 +3,35 @@ body
 {
     margin:             0;
     overflow-x:         hidden;
-
-    margin-top:         30px;
+    font-family: Helvetica, Arial, sans-serif;
+    background: black;
 }
 
 .nobr
 {
     white-space:        nowrap;
 }
 
+input[type="button"] {
+  cursor: pointer;
```

Comment:
Super minor: the rest of this file has 4-space indentation. While I don't care about 2 vs 4, it would be nice to keep it consistent one way or another. :smile: